unstructured/file_utils/filetype.py

```diff
@@ -251,68 +258,60 @@ def detect_filetype(
 
     """Mime type special cases."""
     # third check (mime_type)
-    # NOTE(crag): for older versions of the OS libmagic package, such as is currently
-    # installed on the Unstructured docker image, .json files resolve to "text/plain"
-    # rather than "application/json". this corrects for that case.
-    if mime_type == "text/plain" and extension == ".json":
-        return FileType.JSON
 
     # NOTE(Crag): older magic lib does not differentiate between xls and doc
     if mime_type == "application/msword" and extension == ".xls":
         return FileType.XLS
 
     elif mime_type.endswith("xml"):
-        if extension and (extension == ".html" or extension == ".htm"):
+        if extension == ".html" or extension == ".htm":
             return FileType.HTML
         else:
             return FileType.XML
 
     elif mime_type in TXT_MIME_TYPES or mime_type.startswith("text"):
-        if extension and extension == ".eml":
```

Comment:
Is this taken care of elsewhere now?

Reply:
Yes. It's handled here now. I noticed that it, the .rst and .json checks were functionally the same as a later group of checks (and was prompted to refactor the multiple if/elifs to something else by `make tidy`):
```
if extension in [".eml", ".md", ".rtf", ".html", ".rst", ".tsv", ".json"]:
            return EXT_TO_FILETYPE.get(extension)
```

Reply:
Gotcha, thanks! Yeah I see it down there. That looks cleaner that the big `if`/`elif` block